oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -51,52 +51,96 @@ Reports come into security@openedx.org
+
+Maintainers are requested to notify the security working group if they would like to adjudicate a vulnerability to a different severity than what was originally triaged. The security working group must accept the maintainer's adjudication, but should comment on any considerations around the adjudication.
+
+Maintainers will be reminded to remediate disclosures in proportion to the severity of the disclosure. The following table shows the reminder frequency until resolution of each severity classification:
+
+========    =====   ==================
+Severity    Score   Reminder frequency
+========    =====   ==================
+Low         ≥0.1    Once a semester
+Medium      ≥4.0    Once a quarter
+High        ≥7.0    Once a month
+Critical    ≥9.0    Once a week
+========    =====   ==================
+
+The on-call security working group member is responsible for sending reminders for any vulnerability reminders due during the time they are on-call.
 
 .. note::
 
-   We would also like to specifically note that if an on-call week is particularly time intensive, the on-call member can
-   ask the other working group members for support.
+    Maintainers are encouraged to reach out to security@openedx.org if they need any help remediating security issues.
+
+.. _GitHub repository security advisory: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/creating-a-repository-security-advisory
+
+Details of security vulnerabilities are private until a fix is released
+-----------------------------------------------------------------------
+Members of the Open edX community with access to the details of a disclosure are asked to keep those details private until a fix is publically released for the vulnerability.
+
+Maintainers should use GitHub's `temporary private fork`_ feature within the GitHub repository security advisory created by the security working group to keep the implementation details of a fix for a vulnerability private.
+
+.. _temporary private fork: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability
+
+Operators are ultimately responsible for the security of their Open edX instance
+--------------------------------------------------------------------------------
+The security working group will put forth good faith effort to responsibly forward disclosures related to the operation of a specific Open edX instance to the relevant operator.
+
+Operators wishing to register a desired primary point of contact for security-related matters should email security@openedx.org with the name, domain, contact information for an individual point of contact for security, and an email address to which security reports should be forwarded.
```

Comment:
```suggestion
Operators wishing to register a desired primary point of contact for security-related matters should email security@openedx.org with the name, domain, contact information for an individual point of contact for security, and an email address to which security reports should be forwarded.  We recommend that you provide a consistent alias for this rather than the e-mail address of a specific person to keep the process resilient.  e.g. ``security@companyx.com``
```

Reply:
Since I deleted this section per your previous comment, I moved the spirit of this change to the Guidance for Operators section. Let me know what you think and please feel free to propose other edits.